recipes/ctk-cli/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "ctk-cli" %}
+{%set version = "1.4" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "80bad5eae3ce0d1ece63b53dd2c3dbe4b0bb7f6a18d13748866ae09c83b844b9" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - ctk_cli
+
+about:
+  home: https://github.com/commontk/ctk-cli
+  # license_file: No manifest - see https://github.com/commontk/ctk-cli/pull/16
```

Comment:
I can do this, but this seems to be a general change from how we've handled things in the past, so please note my grousing.